Test on Windows are failing

Not sure yet what this is about:

```
In a future release, GeoPandas will switch to using Shapely by default. If you are using PyGEOS directly (calling PyGEOS functions on geometries from GeoPandas), this will then stop working and you are encouraged to migrate from PyGEOS to Shapely 2.0 (https://shapely.readthedocs.io/en/latest/migration_pygeos.html).
  import geopandas as gpd
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "D:\a\trackintel\trackintel\trackintel\__init__.py", line 1, in <module>
    import trackintel.model
  File "D:\a\trackintel\trackintel\trackintel\model\__init__.py", line 1, in <module>
    from .positionfixes import PositionfixesAccessor
  File "D:\a\trackintel\trackintel\trackintel\model\positionfixes.py", line 8, in <module>
    from trackintel.visualization.positionfixes import plot_positionfixes
  File "D:\a\trackintel\trackintel\trackintel\visualization\__init__.py", line 1, in <module>
    from .locations import plot_locations
  File "D:\a\trackintel\trackintel\trackintel\visualization\locations.py", line 6, in <module>
    from trackintel.visualization.osm import plot_osm_streets
  File "D:\a\trackintel\trackintel\trackintel\visualization\osm.py", line 3, in <module>
    import osmnx as ox
  File "C:\Miniconda3\envs\test\lib\site-packages\osmnx\__init__.py", line 3, in <module>
    from ._api import *
  File "C:\Miniconda3\envs\test\lib\site-packages\osmnx\_api.py", line 17, in <module>
    from .geometries import geometries_from_address
  File "C:\Miniconda3\envs\test\lib\site-packages\osmnx\geometries.py", line 21, in <module>
    from shapely.geos import TopologicalError
ImportError: cannot import name 'TopologicalError' from 'shapely.geos' (C:\Miniconda3\envs\test\lib\site-packages\shapely\geos.py)
Error: Process completed with exit code 1.
```

https://github.com/mie-lab/trackintel/actions/runs/3679511054/jobs/6224048994

--- a/.github/envs/39-latest-conda-forge.yaml
+++ b/.github/envs/39-latest-conda-forge.yaml
@@ -19,7 +19,6 @@ dependencies:
 - similaritymeasures
 - jupyter
 - geopandas>=0.10.0
-- shapely<=1.8.5
 - pygeos>=0.10.0
 
 # tests

--- a/environment.yml
+++ b/environment.yml
@@ -6,7 +6,6 @@ dependencies:
 - python
 - numpy
 - matplotlib
-- pandas
 - geopandas>=0.10.0
 - scikit-learn
 - networkx 

--- a/environment.yml
+++ b/environment.yml
@@ -18,4 +18,3 @@ dependencies:
 - tqdm
 - similaritymeasures
 - pygeos>=0.10.0
-- shapely<=1.8.5

--- a/requirements.txt
+++ b/requirements.txt
@@ -1,6 +1,5 @@
 numpy
 matplotlib
-pandas
 geopandas>=0.10.0
 scikit-learn
 networkx 
@@ -12,4 +11,3 @@ tqdm
 similaritymeasures
 pygeos>=0.10.0
 virtualenv
-pytest

--- a/requirements.txt
+++ b/requirements.txt
@@ -13,4 +13,3 @@ similaritymeasures
 pygeos>=0.10.0
 virtualenv
 pytest
-shapely<=1.8.5

